qmi/instruments/newport/tlb670x.py

```diff
@@ -114,7 +114,7 @@ def _send(self, command_string: str) -> str:
             if len(response) > 2 and len(response[-2]) > 0 and response[-2] != "OK":
                 # The response probably is this. This presumes the last value in list is [''] due to "...\r\n" split.
                 # But check for sporadic "*IDN?" query response first.
-                idn_match = re.search("v\d.\d(.\d)? [\d/]+ SN[\d]+", response[-2])
```

Comment:
no update to tests to verify this?

Reply:
It was a warning that in the _future_ the regular string won't be accepted, but it is still accepted. This is only to get rid of that deprecation warning. Therefore there is no actual change in a unit-test that tests this line.

Out of boy-scouting inspiration, I did add unit-tests for testing the `reset` RPC command, which was actually not tested until now.